src/library_webgl.js

```diff
@@ -626,7 +626,8 @@ var LibraryGL = {
       if (Module['preinitializedWebGLContext']) {
         var ctx = Module['preinitializedWebGLContext'];
 #if MAX_WEBGL_VERSION >= 2
-        webGLContextAttributes.majorVersion = (typeof WebGL2RenderingContext != 'undefined' && ctx instanceof WebGL2RenderingContext) ? 2 : 1;
+        // The ctx object may not be of a known class (e.g. it may be a debug wrapper), so we ask it for its version instead.
```

Comment:
```suggestion
        // The ctx object may not be of a known class (e.g. it may be a debug wrapper), so we ask it for its version rather than use instanceof.
```